packages/embed-grid/src/App.tsx

```diff
@@ -0,0 +1,180 @@
+  const [isLoading, setIsLoading] = useState(true);
+  const searchParams = useMemo(
+    () => new URLSearchParams(window.location.search),
+    []
+  );
+  const canCopy = searchParams.get('canCopy') != null;
+  const canDownloadCsv = searchParams.get('canDownloadCsv') != null;
+
+  const initApp = useCallback(async () => {
+    try {
+      // Connect to the Web API server
+      const baseUrl = new URL(
+        process.env.REACT_APP_CORE_API_URL ?? '',
+        `${window.location}`
+      );
+
+      const websocketUrl = `${baseUrl.protocol}//${baseUrl.host}`;
+
+      log.debug(`Starting connection...`);
+      const connection = new dh.IdeConnection(websocketUrl);
+
+      // Start a code session. For this example, we use python.
+      log.debug(`Starting session...`);
+      const session = await connection.startSession('python');
+
+      // Get the table name from the query param `name`.
+      const name = searchParams.get('name');
+
+      if (!name) {
+        throw new Error('No name param provided');
+      }
+
+      log.debug('Loading table', name, '...');
+
+      // Load the table up.
+      const table = await loadTable(session, name);
+
+      // Create the `IrisGridModel` for use with the `IrisGrid` component
+      log.debug(`Creating model...`);
+
+      const newModel = await IrisGridModelFactory.makeModel(table);
+
+      setModel(newModel);
+
+      log.debug('Table successfully loaded!');
+    } catch (e: unknown) {
+      log.error('Unable to load table', e);
+      setError(`${e}`);
+    }
+    setIsLoading(false);
+  }, [searchParams]);
+
+  useEffect(() => {
+    initApp();
+  }, [initApp]);
+
+  useEffect(() => {
+    function receiveMessage(e: MessageEvent) {
+      const { command, value } = e.data as { command: Command; value: unknown };
+      switch (command) {
+        case 'filter': {
+          const filterCommandValues = value as FilterCommandType[];
+          const newInputFilters = filterCommandValues.map(
+            ({ name, value: filterValue }) => {
+              const column = model?.columns.find(c => c.name === name);
+              if (column == null) {
+                throw new Error(`Could not find column named ${name}`);
+              }
+              return {
+                name,
+                value: filterValue,
+                type: TableUtils.getNormalizedType(column.type),
+              };
+            }
+          );
+          setInputFilters(newInputFilters);
+          break;
+        }
+        case 'sort': {
+          const sortCommandValues = value as SortCommandType[];
+          const newSorts = sortCommandValues.map(({ name, direction }) => {
+            const column = model?.columns.find(c => c.name === name);
+            if (column == null) {
+              throw new Error(`Could not find column named ${name}`);
+            }
+            return direction === 'DESC'
+              ? column.sort().asc()
+              : column.sort().desc();
```

Comment:
```suggestion
            return direction === 'DESC'
              ? column.sort().desc()
              : column.sort().asc();
```
Should it be the other way around?